templates/bulk-actor-delete-confirmation.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Bulk ActivityPub actor deletion confirmation template.
+ *
+ * @package Activitypub
+ */
+
+use Activitypub\Collection\Actors;
+
+/* @var array $args Template arguments. */
+$args = wp_parse_args( $args ?? array() );
+
+$users     = $args['users'] ?? array();
+$send_back = $args['send_back'] ?? '';
+
+// Validate users.
+if ( empty( $users ) ) {
+	wp_die( esc_html__( 'No users selected.', 'activitypub' ), '', array( 'back_link' => true ) );
+}
+
+// Prepare user data for display.
+$users = \get_users( array( 'include' => $users ) );
+
+// If no users with ActivityPub capability, redirect back.
+if ( ! $users ) {
+	wp_safe_redirect( $send_back );
+	exit;
+}
+
+require_once ABSPATH . 'wp-admin/admin-header.php';
+?>
+<div class="wrap">
+	<h1><?php esc_html_e( 'Delete Users from Fediverse', 'activitypub' ); ?></h1>
+	<p><?php esc_html_e( 'You&#8217;ve just removed the capability to publish to the Fediverse for the selected users, do you want to also remove them from the Fediverse?', 'activitypub' ); ?></p>
+	<p><?php echo wp_kses( __( 'Fediverse deletion is optional but recommended to properly notify your followers, <strong>but be aware that this action might be irreversible.</strong>', 'activitypub' ), array( 'strong' => array() ) ); ?></p>
```

Comment:
```suggestion
	<p><?php echo wp_kses( __( 'Fediverse deletion is optional but recommended to properly notify your followers. <strong>This action is irreversible.</strong>', 'activitypub' ), array( 'strong' => array() ) ); ?></p>
```
There are two "but"s in this sentence, let's simplify it.